modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -114,14 +114,7 @@ Local Couchbase Server::
 -- 
 [source,java]
 ----
-include::example$StartUsing.java[tag=connect,indent=0]
-
-include::example$StartUsing.java[tag=bucket,indent=0]
-
-include::example$StartUsing.java[tag=collection,indent=0]
-
-include::example$StartUsing.java[tag=upsert,indent=0]
-
+include::example$StartUsing.java
```

Comment:
oh wait I think you may need `[]` at the end to see the full example.

Reply:
and `[tags=**]` will strip the antora tag declarations

Reply:
yep ^ that should do it